pom.xml

```diff
@@ -267,6 +268,20 @@
             <plugin>
                 <groupId>org.apache.rat</groupId>
                 <artifactId>apache-rat-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>rat-check</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <excludes>
+                                <exclude>*.md</exclude>
```

Comment:
why? Just add a license in HTML comment <-- -->

Reply:
Discussed with @tivv offline, this should be ok as we should be excluding markdown files.